src/main/java/com/iota/iri/service/TipsManager.java

```diff
@@ -25,20 +25,20 @@
     private final boolean testnet;
     private final int milestoneStartIndex;
 
-    private int RATING_THRESHOLD = 75; // Must be in [0..100] range
+    private int ratingThreshold = 75; // Must be in [0..100] range
```

Comment:
The conflict is just on this line, it's an easy fix. Let's merge this PR after the fix.